app/workers/bgs/submit_form674_job.rb

```diff
@@ -9,37 +9,48 @@ class Invalid674Claim < StandardError; end
     include Sidekiq::Worker
     include SentryLogging
 
-    sidekiq_options retry: false
+    attr_reader :claim, :user
+
+    sidekiq_options retry: 14
+
+    sidekiq_retries_exhausted do
+      salvage_save_in_progress_form(FORM_ID, @user_uuid, @in_progress_copy)
+      DependentsApplicationFailureMailer.build(user).deliver_now if user&.email.present?
```

Comment:
I think this is tickets out of sync, shouldnt we be leveraging the backup submission process here?

Reply:
Yes, it should be updated to reflect the backup submission now